MIP40/MIP40c3-Subproposals/MIP40c3-SP42.md

```diff
@@ -103,31 +107,37 @@ The budget cap will differ from the actual expenses of the core unit. The cap re
 
 The initial seed transaction covers the contingency buffers for the core unit:
 
-* Three months of regular expenses: $122,187.50
+* Three months of regular expenses: $470,639.8
 * The yearly and one-time costs included in the budget: $2,530 + $70,725 = $73,255
-* Total for three months of runway: $122,187.50 + $73,255 = $195,442.50
+* Total for three months of runway: $470,639.8 + $73,255 = $543,894.8
 
 The streaming transaction covers ongoing activity for the core unit:
 
-* Total remaining costs for 12 months: $122,187.50 x 3 = $366,562.50
+* Total remaining costs for 12 months: $470,639.8 x 3 = $1,411,919.4
```

Comment:
Is this mentioned in the subproposal? Seems like a larger change than is intended from the text

Reply:
Check [MIP4c3](https://forum.makerdao.com/t/mip4c3-xx-rapid-payment-for-small-bug-bounties/12869). The point is to pre-allocate some DAI to pay for bug bounties. This DAI won't be spent unless bugs are reported.